src/tool/subcommands/api_cmd.rs

```diff
@@ -1108,6 +1108,7 @@ fn eth_tests() -> Vec<RpcTest> {
             .unwrap(),
         ),
         RpcTest::basic(Web3ClientVersion::request(()).unwrap()),
+        RpcTest::basic(EthMaxPriorityFeePerGas::request(()).unwrap()),
```

Comment:
Can we use `identity`? Do we estimate a different gas fee than Lotus?

Reply:
no it should be `identity`, fixed

Reply:
`EthMaxPriorityFeePerGas` rpc uses `estimate_gas_premium` fn which has a randomness component
https://github.com/ChainSafe/forest/blob/ffa93120d61712b3bbdd6d7e16c8f049549906cb/src/rpc/methods/gas.rs#L151-L154
because of this identity test cannot be used here